sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/AnalyzeTextOptions.java

```diff
@@ -45,7 +45,7 @@
      * 'Simple', 'Stop', 'Whitespace'
      */
     @JsonProperty(value = "analyzer")
-    private LexicalAnalyzerName analyzer;
+    private LexicalAnalyzerName analyzerName;
```

Comment:
Two questions.

1. Should this and `tokenizerName` be `final` and only set in the constructor?
2. If we choose not to do #1 should the other value be set to null when using `setAnalyzerName` or `setTokenizerName`? Based on other code it appears these should be mutually exclusive.

Reply:
Two ctors are using different ones, so option 2 is doable. I will make the change